karton/config_extractor/config_extractor.py

```diff
@@ -70,20 +71,58 @@ def args_parser(cls):
             help="Malduck extractor modules directory",
             default="extractor/modules",
         )
+        parser.add_argument(
+            "--tag",
+            help="Add specified tag to all produced configs",
+            default=[],
+            nargs="+",
+        )
+        parser.add_argument(
+            "--attribute",
+            help="Add specified attribute to all produced configs (format: key=value)",
+            default=[],
+            nargs="+",
+        )
         return parser
 
     @classmethod
     def main(cls):
         parser = cls.args_parser()
         args = parser.parse_args()
 
+        attributes: DefaultDict[str, List[str]] = defaultdict(list)
+        for attr in args.attribute:
+            key, value = attr.split("=")
```

Comment:
```suggestion
            key, value = attr.split("=", 1)
```
Would this be crazy?